src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -26,6 +26,11 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(RefOnlyMicrosoftBuildFrameworkVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
+
+    <!-- These PackageReferences aren't needed to make our build work, but ensure correct NuGet packaging. We get references to these packages because
+         we have a ProjectReference to BuildHost.csproj, but since it's PrivateAssets="true" they don't flow into this package's package references when packaging. -->
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsLoggingAbstractionsVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
really not happy about newtonsoft dep either.  we're trying to heavily get off of it (both ourselves and razor) since it's so heavyweight.

Reply:
This will be addressed as a mop-up -- today the use of StreamJsonRpc was already using Newtonsoft.Json for marshalling, so it requires a bit of care to ensure the JSON types we were using today will still work and we don't break any other serialization if there are shared types involved. I don't think it's too bad to fix but that'll be a follow-up.